src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -32,27 +33,31 @@ public class CommandTestUtil {
     public static final String VALID_EMAIL_BOB = "bob@example.com";
-    public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
-    public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_CLASS_GROUP_AMY = "B01";
+    public static final String VALID_CLASS_GROUP_BOB = "F14";
+    public static final String VALID_TELEGRAM_AMY = "@amybee";
+    public static final String VALID_TELEGRAM_BOB = "@bobchoo";
+    public static final String VALID_GITHUB_AMY = "amybee";
+    public static final String VALID_GITHUB_BOB = "bobchoo";
 
     public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
     public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
     public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
     public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
     public static final String EMAIL_DESC_AMY = " " + PREFIX_EMAIL + VALID_EMAIL_AMY;
     public static final String EMAIL_DESC_BOB = " " + PREFIX_EMAIL + VALID_EMAIL_BOB;
-    public static final String ADDRESS_DESC_AMY = " " + PREFIX_ADDRESS + VALID_ADDRESS_AMY;
-    public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
+    public static final String CLASS_GROUP_DESC_AMY = " " + PREFIX_CLASS_GROUP + VALID_CLASS_GROUP_AMY;
+    public static final String CLASS_GROUP_DESC_BOB = " " + PREFIX_CLASS_GROUP + VALID_CLASS_GROUP_BOB;
+    public static final String TELEGRAM_DESC_AMY = " " + PREFIX_TELEGRAM + VALID_TELEGRAM_AMY;
+    public static final String TELEGRAM_DESC_BOB = " " + PREFIX_TELEGRAM + VALID_TELEGRAM_BOB;
+    public static final String GITHUB_DESC_AMY = " " + PREFIX_GITHUB + VALID_GITHUB_AMY;
+    public static final String GITHUB_DESC_BOB = " " + PREFIX_GITHUB + VALID_GITHUB_BOB;
 
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
-    public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
-    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_CLASS_GROUP_DESC = " " + PREFIX_CLASS_GROUP; // empty
+    public static final String INVALID_TELEGRAM_DESC = " " + PREFIX_TELEGRAM + "hi"; // '*' not allowed in tags
```

Comment:
prefix_telegram is "/t" i.e. the prefix the user enters to indicate to the program that they are adding a tele ID. it is an invalid description because tele ID should start with "@" symbol (we can change this if its not suited to our purpose though)

Reply:
Ohhh I see, my bad. Thanks